lib/logproto/logproto-text-server.c

```diff
@@ -169,21 +169,36 @@ log_proto_text_server_extract(LogProtoTextServer *self, LogProtoBufferedServerSt
   return FALSE;
 }
 
+static inline gboolean
+_is_message_boundary(gchar ch)
+{
+  if (ch <= 13)
+    {
+      /* set bits to 1 where they are considered to be message boundaries,
+       * CR, LF and NUL characters are set */
+      const guint32 ch_bits = 0x2401;
```

Comment:
[nit]
I think it helps with readability:
```suggestion
      const guint32 ch_bits = (1 << '\0') | (1 << '\n') | (1 << '\r');
```

Compilers generate the same instructions:
https://godbolt.org/z/We6MjGzWh